### PR TITLE
Add hide recommendations button on mobile

### LINKED ISSUE #426: Button to hide recommended videos on mobile
Title.

### PR DESCRIPTION
**Pull request by DeeHechEss**
Closes #426 

I added a button similar to the hide description which is shown only on the single column grid (mobile) view to toggle the recommendations. I also added some logic to automatically display the recommendations if the screen increases in size.

Contribution for Hacktoberfest :)

### CODE REVIEW

File: src/components/WatchVideo.vue [hunk 1]
```diff
@@ -144,6 +149,7 @@ import Comment from "@/components/Comment.vue";
 export default {
     name: "App",
     data() {
+        const smallViewQuery = window.matchMedia("(max-width: 640px)");
```
Comment (FireMasterK) [inline src/components/WatchVideo.vue:152]: Please move this to a computed variable.

**Review by FireMasterK — CHANGES_REQUESTED** [review]

File: src/components/WatchVideo.vue (hunk 1 above)
Comment (DeeHechEss) [reply src/components/WatchVideo.vue]: I tried as you suggested but I also had to initialize smallView to a boolean as it seems you can't access computed properties in data definition. Let me know if you had another way in mind.

Comment (FireMasterK) [reply src/components/WatchVideo.vue]: I had thought about removing `smallViewQuery` and replacing it with `smallView` as a computed variable!

Comment (DeeHechEss) [reply src/components/WatchVideo.vue]: I tried that but it wouldn't update itself when I resized the window. I could use forceUpdate in the event listener instead though.

`    computed: {
        smallView() {
            return window.matchMedia("(max-width: 640px)").matches;
        }
    },`

Comment (FireMasterK) [reply src/components/WatchVideo.vue]: Oh okay, then your latest commit was unnecessary then, I'll revert it and merge this!

**Review by FireMasterK — APPROVED** [review]

**Pull request merged**